deps/v8/gypfiles/v8.gyp

```diff
@@ -2508,41 +2509,31 @@
       },
     }, # v8_libsampler
     {
-      'target_name': 'js2c',
+      'target_name': 'js2c_extras',
       'type': 'none',
       'conditions': [
         ['want_separate_host_toolset==1', {
-          'toolsets': ['host'],
+          'toolsets': [ 'host', 'target', ],
         }, {
-          'toolsets': ['target'],
+          'toolsets': [ 'target', ],
         }],
       ],
-      'variables': {
-        'library_files': [
-          '../src/js/macros.py',
-          '../src/message-template.h',
-          '../src/js/prologue.js',
-        ],
-        'libraries_extras_bin_file': '<(SHARED_INTERMEDIATE_DIR)/libraries-extras.bin',
-      },
       'actions': [
        {
           'action_name': 'js2c_extras',
           'inputs': [
             '../tools/js2c.py',
             '<@(v8_extra_library_files)',
           ],
-          'outputs': ['<(SHARED_INTERMEDIATE_DIR)/extras-libraries.cc'],
+          'outputs': [ '<(SHARED_INTERMEDIATE_DIR)/extras-libraries.cc', ],
```

Comment:
I don't really care for such whitespace changes, they probably just make back-ports more difficult. They're also not applied consistently throughout (see the next file.)

Reply:
I'm not sure backport is a concern for this file, but I agree it's unnecessary churn, and is inconsistent with surrounding.
Reverted.